### PR TITLE
Fix test vtimer msg

### PR DESCRIPTION
**Pull request by mehlis**

### CODE REVIEW

File: test_vtimer_msg/Makefile [hunk 1]
```diff
@@ -21,6 +21,7 @@ export RIOTBASE = $(CURDIR)/../../RIOT
 ## Modules to include. 
 
 USEMODULE += auto_init
+USEMODULE += hwtimer
```
Comment (LudwigKnuepfer) [inline test_vtimer_msg/Makefile]: vtimer depends on hwtimer already

File: test_vtimer_msg/main.c [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #include "thread.h"
 #include "msg.h"
 
-char timer_stack[KERNEL_CONF_STACKSIZE_DEFAULT];
+char timer_stack[KERNEL_CONF_STACKSIZE_PRINTF];
```
Comment (LudwigKnuepfer) [inline test_vtimer_msg/main.c:7]: how about `KERNEL_CONF_STACKSIZE_MAIN`?

Comment (LudwigKnuepfer) [inline test_vtimer_msg/main.c:7]: On second thought - it's not that big...

**Comment by mehlis** [comment]
@LudwigOrtmann updated: to address your comment concerning the hwtimer include: removed commit

**Comment by LudwigKnuepfer** [comment]
ACK

**Pull request merged**